example/example/spiders/books_04_overrides_03.py

```diff
@@ -21,9 +21,8 @@ def book_urls(self):
         return []
 
 
-class BookPage(ItemWebPage):
-    def to_item(self):
-        return None
+class BookPage(WebPage):
+    ...
```

Comment:
What's the reason of using `...` in the example, not `pass`? I'd expect `...` to appear when the example is incomplete, and the implementation is missing, as an alternative to `# ...` comment. 

But here the intention is different: empty BookPage class is a complete definition, it's a "generic" page object which is only there to be overridden with website-specific page objects, it doesn't need an implementation.

Reply:
I've used `...` mostly as a matter of style. I've changed this to `pass` instead. https://github.com/scrapinghub/scrapy-poet/pull/89/commits/bfb237b434a840dfdedcc1e2c2d8c7c35e610b57